manifests/params.pp

```diff
@@ -86,14 +86,17 @@
   $service_ensure          = 'running'
   $service_restart         = undef
 
-  $ss_version              = '1.14.11'
-  $ss_install_base         = '/var/simplesamlphp'
+  $ss_version              = '1.15.4'
+  $ss_install_base         = '/var/www/simplesamlphp'
+  $ss_sp_name              = 'shibvm-sp'
   $ss_sp_host              = undef
   $ss_sp_port              = '31443'
   $ss_sp_url_path          = 'simplesaml'
+  $ss_manage_cert          = true
   $ss_admin_password       = undef
   $ss_secret_salt          = undef
   $ss_cert_owner           = undef
   $ss_cert_group           = undef
+  $ss_cert_dir             = '/var/www/simplesamlphp/cert'
```

Comment:
Is this inside a docroot? Is it served up by apache?

Reply:
This is the default for SimpleSamlPHP. These key pairs are used for signing and encrypting between the SP and the IdP. I tried using the host's cert, but since the PHP scripts run as a privileged user, it was unable to read the private key. Given that these public and private keys must readable by the web server user, it makes more sense to keep them separate.